spyder/plugins/editor/lsp/client.py

```diff
@@ -151,10 +151,11 @@ def start(self):
                 self.transport_args += ['--server-log-file', server_log_file]
 
             # Start server with logging options
-            if get_debug_level() == 2:
-                self.server_args.append('-v')
-            elif get_debug_level() == 3:
-                self.server_args.append('-vv')
+            if self.language != 'cpp':
```

Comment:
This needs to change too, according to my suggestion of using `C/C++` for this language name.

Reply:
The pygments language entry for C/C++ is Cpp, that is the way in which we manage the mapping between files and servers. We could add some sort of intermediate proxy to map simply the name on the configuration screen

Reply:
Ok, I didn't know that. Then let's leave that for later because it's not related to this PR.